openedx/core/djangoapps/geoinfo/middleware.py

```diff
@@ -31,13 +30,13 @@ def process_request(self, request):
 
         Store country code in session.
         """
-        new_ip_address = get_real_ip(request)
+        new_ip_address, _ = get_client_ip(request)
```

Comment:
```suggestion
        new_ip_address = get_client_ip(request)[0]
```
Let's keep usage consistent unless there is a reason not to.

Reply:
Done.